package.json

```diff
@@ -21,6 +21,7 @@
     "aws-xray-sdk-core": "3.5.0",
     "aws-xray-sdk-express": "3.5.0",
     "chai": "^4.2.0",
+    "chai-as-promised": "^7.1.1",
```

Comment:
Following up to the comment on moving to sdk_contrib, the dependencies that are unique to this instrumentation would then be placed in the package.json of the new instrumentation package in sdk_contrib. 
You can reference this [fastify instrumentation PR](https://github.com/aws/aws-xray-sdk-node/pull/523) as a guide to help